app/Http/Controllers/Traits/TransformsRequests.php

```diff
@@ -117,8 +117,13 @@ public function paginatedCollection($query, $request, $code = 200, $meta = [], $
         $resource->setPaginator(new IlluminatePaginatorAdapter($paginator));
 
         $includes = null;
+
         if ($request->query('include')) {
-            $includes = explode(',', $request->query('include'));
+            if (is_array($request->query('include'))) {
```

Comment:
We can simplify this conditional a bit, since the "special" case is if the query is a CSV. You should also be able to remove the wrapping `if ($request->query('include')) {` since it should return null if not provided.

```php
$includes = $request->query('include');

if (is_string($includes)) {
    $includes = explode(',', $request->query('include'));
}
```

Reply:
ah yep haha this is much better.